Python/generated_cases.c.h

```diff
@@ -464,22 +430,21 @@
         }
 
         TARGET(LIST_APPEND) {
-            PyObject *v = POP();
-            PyObject *list = PEEK(oparg);
-            if (_PyList_AppendTakeRef((PyListObject *)list, v) < 0)
-                goto error;
+            PyObject *v = PEEK(1);
+            PyObject *list = PEEK(oparg + 1);  // +1 to account for v staying on stack
+            if (_PyList_AppendTakeRef((PyListObject *)list, v) < 0) goto pop_1_error;
```

Comment:
Any plans to make these conform to PEP 7 (braces and newline for body)?

I know this isn't exactly meant for human consumption, but as a human who will definitely be consuming this output, I think it might be worth improving the readability of the generated code a bit.

Reply:
Is the issue that you easily miss the `goto`, or is your brain just yelling "missing braces" at you? I intentionally put it all on one line so the reason why PEP 7 requires braces doesn't apply. I think there's also a technical issue where we suddenly need to know the indentation level (but that's recovered easily enough).

All in all my preferred answer is "no". :-) But I feel you. My brain yells at me a little bit too.